
Training materials for the methylation course (bisulfite-sequencing) on **Monday 23rd November** as part of the **[International School on Big Data and Epigenomics applied to Public Health](https://campusvirtual.fiocruz.br/gestordecursos/hotsite/BigDataEpigenomics)**


### Credits

All the training materials have been obtained from the following resources:

- Babraham Institute's [Analysing bisulfite methylation sequencing data course](https://www.bioinformatics.babraham.ac.uk/training.html#bsseq)
- Biconductor edgeR package section 4.7 [Bisulfite sequencing of mouse oocytes](http://bioconductor.org/packages/release/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf)


### Introduction

Provide a look at the computational steps needed for the analysis of bisulfite sequencing data.
Lectures and practicals covering the fundamentals of DNA methylation in genomes, processing of raw sequence data and  QC, visualisation, quantification and differential methylation analysis.


### Schedule

Time | Session | Link
-----|---------|------
9:00-9:40 | Lecture | [Introduction to methylation analysis. Data processing, QC and alignment](lectures/Introduction.pdf)
9:40-10:10 | Practical | [QC and Mapping of BS-Seq data](practicals/QC_mapping.pdf)
10:10-10:20 | Recap | 
10:20-10:40 | Coffee break | 
10:40-11:20 | Lecture | [Visualising and exploring methylation data](lectures/Visualisation.pdf)
10:40-11:20 | Lecture | [Differential methylation analysis](lectures/Differential_Methylation_Analysis.pdf)
11:20-11:50 | Practical | [Differential methylation analysis with edgeR](practicals/Differential_Methylation_Analysis_edgeR.pdf)
11:50-12:00 | Recap |


### Comments / Feedback

Please get in touch with Sergio Martinez Cuesta: sermarcue [at] gmail.com


